.github/workflows/tests.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
       - name: Install conda dependencies
         run: |
           mamba install datacube gdal pycodestyle pylint pytest-cov pytest numexpr lxml pydash python-rapidjson \
-          lxml pydash python-rapidjson ruamel.yaml structlog ciso8601 cattrs boltons dawg
+          lxml pydash python-rapidjson ruamel.yaml structlog ciso8601 cattrs boltons dawg platformdirs=2.4.0
```

Comment:
Are we using platformdirs?

Reply:
pylint is using it, since `2.4.1`, the support for python `3.6` has been dropped, which caused the pipeline broken, hence pin the version. If we drop `3.6` support, there no need then.